test/Orleans.Providers.Legacy.Test/Orleans.Providers.Legacy.Test.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Providers.Legacy.Tests</PackageId>
```

Comment:
Should not have PackageId and have IsPackable false.
Also, name of the folder and the project should be `.Tests` (not `.Test`)